<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZEDMainWindow</class>
 <widget class="QMainWindow" name="ZEDMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>856</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ZEditor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/ApplicationIcon/ZEDITOR.ico</normaloff>:/ApplicationIcon/ZEDITOR.ico</iconset>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QGridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1307</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </action>
  <action name="actScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </action>
  <action name="actHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="ObjectSellectorOpenAction">
   <property name="text">
    <string>Object Sellector</string>
   </property>
  </action>
  <action name="actOpenConsole">
   <property name="text">
    <string>Console</string>
   </property>
  </action>
  <action name="actOpenAssets">
   <property name="text">
    <string>Asserts</string>
   </property>
  </action>
  <action name="PropertiesWindow_2">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="AboutOpenAction">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="GizmoOpenAction">
   <property name="text">
    <string>Gizmo</string>
   </property>
  </action>
  <action name="actSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="DeleteAction">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actHide">
   <property name="text">
    <string>Hide</string>
   </property>
  </action>
  <action name="actUnhide">
   <property name="text">
    <string>Unhide</string>
   </property>
  </action>
  <action name="actFreeze">
   <property name="text">
    <string>Freeze</string>
   </property>
  </action>
  <action name="actUnfreeze">
   <property name="text">
    <string>Unfreeze</string>
   </property>
  </action>
  <action name="OperationsToolbarOpenAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Operations</string>
   </property>
  </action>
  <action name="EditToolbarOpenAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="FileToolbarOpenAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="SaveAsAction">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actClone">
   <property name="text">
    <string>Clone</string>
   </property>
  </action>
  <action name="CutAction">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="PasteAction">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionIndex">
   <property name="text">
    <string>Index</string>
   </property>
  </action>
  <action name="actionTutorials">
   <property name="text">
    <string>Tutorials</string>
   </property>
  </action>
  <action name="actionEngine_Web">
   <property name="text">
    <string>Engine Web</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actWireframe">
   <property name="text">
    <string>Wireframe</string>
   </property>
  </action>
  <action name="actMap">
   <property name="text">
    <string>Map</string>
   </property>
  </action>
  <action name="actEntities">
   <property name="text">
    <string>Entities</string>
   </property>
  </action>
  <action name="actSkybox">
   <property name="text">
    <string>Skybox</string>
   </property>
  </action>
  <action name="actOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actRenderType">
   <property name="text">
    <string>Render Type</string>
   </property>
  </action>
  <action name="actGoToEntity">
   <property name="text">
    <string>Go To Entity</string>
   </property>
  </action>
  <action name="Assert_BrowserOpenAction">
   <property name="text">
    <string>Assert Browser</string>
   </property>
  </action>
  <action name="actionLoad_Environment">
   <property name="text">
    <string>Load Environment</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="QuickSelectorOpenAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Quick Selector</string>
   </property>
  </action>
  <action name="actUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="SceneListOpenAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Scene List</string>
   </property>
  </action>
  <action name="actGridToogle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered(bool)</signal>
   <receiver>ZEDMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
